### ISSUE TITLE
Send a Message every 12 Hours.

### ISSUE THREAD

**Issue by BorderBoy** [tgalal/yowsup#1241]
Yowsup sometimes crashes after days of working because of several reasons. Unfortunately, I can't fix them. But is there a way to send a default message every 12 hours, so I notice the program has crashed?

**Comment by bdbais** [comment]
I can't understand you, Are you asking to send a message everytime it crash, simple send a message when it restart, so you will see how many times whatsapp server log you out :) or your program crash, but remember don't spam, so put a counter in your program and after 2-3 times for day stop sending init message.

**Comment by BorderBoy** [comment]
No. I can't manage to restart when it Crashes. I only want to Send Message every 12 Hours

**Comment by e0xify** [comment]
create a cronjob and use screen / external command

**Comment by bdbais** [comment]
cronjob on linux e scheduled task on windows.
